kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -176,6 +176,18 @@ data:
     [agent_settings.high_log_scale]
       enabled = false
 
+    # Retina Network Flow Logs throttle settings
+    # Controls the rate at which network flow log messages are processed. 
+    # throttle_rate: Amount of messages for the time.
```

Comment:
remove top level comments and have them inline.

Reply:
addressed